capnpc/rust.capnp

```diff
@@ -27,6 +27,32 @@ annotation parentModule @0xabee386cd1450364 (file) :Text;
 #    }
 #  }
 
+annotation imports @0xc3b9fe42a83105cd (file) :List(Import);
+# Allows specifying that the generated code for an import is located in another
+# crate.
+#
+# You only need this if your code uses imported types whose generated code is
+# in another crate. You can only use this annotation once in all the files
+# compiled together.
+#
+#    using Json = import "/capnp/compat/json.capnp";
+#
+#    $Rust.imports([
+#        (path = "/capnp/compat/json.capnp", crate = "capnp_json")
```

Comment:
It could make sense for the key to be the 64-bit capnp ID of the file, rather than the string `path`. That seems to me like it would maybe be a bit more robust. I'm not sure which alternative would be more user friendly.